pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/QueryOp.java

```diff
@@ -30,13 +38,22 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class QueryOp extends BaseOp {
+  private static final Logger LOGGER = LoggerFactory.getLogger(QueryOp.class);
+
+  private static final String NUM_DOCS_SCANNED = "numDocsScanned";
```

Comment:
Rename it to `NUM_DOCS_SCANNED_KEY`. Same for the following one.

Reply:
done